sdcm/nemesis.py

```diff
@@ -2875,8 +2875,8 @@ def decommission_post_action():
             delay=0
         )
         ParallelObject(objects=[trigger, watcher], timeout=600).call_objects()
-        decommission_post_action()
-        self.target_node.run_nodetool("rebuild")
+        new_node = decommission_post_action()
+        new_node.run_nodetool("rebuild")
```

Comment:
This will likely fail when `decommission_post_action()` returns `None` here: https://github.com/scylladb/scylla-cluster-tests/blob/992dde62c96a31eca00a24f6a77c5fdcd2fdd9bd/sdcm/nemesis.py#L2850-L2854 We can just do a conditional here:
```suggestion
        if new_node:
            new_node.run_nodetool("rebuild")
```

Reply:
in that case, shall be too:
``
else:
    self.log.error('there is no new node, so cannot rebuild')
```

Reply:
> NodeStayInClusterAfterDecommission

In this case we will run rebuild on the `self.target_node` I think

Reply:
changed